### PR TITLE
Cleaner UI for explore page (#1253)

### LINKED ISSUE #1523: Issue render failed.
visit https://try.gitea.io/trygitea123/BigRed/issues/1

### PR DESCRIPTION
**Pull request by pgaskin**
| Old | New |
| - | - |
|||

### CODE REVIEW

**Comment by pgaskin** [comment]
#1253

**Comment by strk** [comment]
Trusted LGTM (trusting the screenshots :)

**Comment by pgaskin** [comment]
@strk Don't worry, I have applied and tested this on my server.

**Comment by bkcsoft** [comment]
For Large screens (that's `col-12-l` in bootstrap FWIW) I'd love to have the search-bar and tab-bar be the same one. But that's just my preference 🙂 Otherwise 👍 LGTM

**Review by appleboy — APPROVED** [review]

File: public/less/_explore.less [hunk 1]
```diff
@@ -3,6 +3,13 @@
 	padding-bottom: @footer-margin * 2;
 
 	.navbar {
+        justify-content: center !important;
```
Comment (tboerger) [inline public/less/_explore.less]: Please don't use important, just be more specific and you can avoid that.

Comment (pgaskin) [reply public/less/_explore.less]: Done

**Comment by pgaskin** [comment]
Finished

**Comment by KangoV** [comment]
Would this look better if the tabs looked the same as tabs on other pages and left aligned? I'm all for a consistent look.

**Comment by pgaskin** [comment]
@KangoV I have tried left aligned and it does not look good.

**Comment by pgaskin** [comment]
Can this be merged now?

**Comment by bkcsoft** [comment]
@geek1011 Mind going through the other pages that have this side-menu now? 😉

**Comment by tboerger** [comment]
@KangoV maybe we should build it like that entirely ;)

**Comment by pgaskin** [comment]
@bkcsoft Do you have a list? :wink:

**Pull request merged**